mindsdb/api/mysql/mysql_proxy/mysql_proxy.py

```diff
@@ -388,7 +390,10 @@ def insert_predictor_answer(self, insert):
                     default_store.delete_datasource(ds_name)
                 raise Exception(f"Column '{col}' not exists")
 
-        mdb.learn(insert['name'], ds, insert['predict'], kwargs)
+        if insert['name'] in [x['name'] for x in custom_models.get_models()]:
+            custom_models.learn(insert['name'], ds, insert['predict'], kwargs)
+        else:
+            mdb.learn(insert['name'], ds, insert['predict'], kwargs)
```

Comment:
I don't like this not obvious behavior. Probably in future we should add 'update' statement for such cases.